ibis/backends/tests/test_impure.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import annotations
+
+import sys
+
+import pytest
+
+import ibis
+import ibis.common.exceptions as com
+from ibis import _
+from ibis.backends.tests.errors import (
+    PsycoPg2InternalError,
+    Py4JJavaError,
+    PyDruidProgrammingError,
+)
+
+tm = pytest.importorskip("pandas.testing")
+
+pytestmark = pytest.mark.xdist_group("impure")
```

Comment:
why is this needed?

Reply:
Concurrent execution of `CREATE OR REPLACE FUNCTION` in postgres doesn't seem to work. This ensures that all tests in this module run in the same process as long as `--dist=loadgroup` is passed, which it is.

Reply:
Makes sense, that seems like something we shouldn't worry about further. Maybe add a comment here? Also fine to not, if someone removes it they will find out from failing tests.

Reply:
I'll add a comment, because I already forgot when revisiting this in review 😅 !